packages/account-v2/src/containers/AccountLimitsContainer/AccountLimitsTable.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
are we going to add test cases for all components? @amina-deriv

Reply:
yes. test cases will be covered in separate PR